docs/production/install.md

```diff
@@ -67,6 +67,22 @@ If the script gives an error, consult [Troubleshooting](#troubleshooting) below.
   If you'd prefer to acquire an SSL certificate yourself in any other
   way, it's easy to [provide it to Zulip][doc-ssl-manual].
 
+* `--postgresql-version`: Sets the version of PostgreSQL that will be installed.
+
+* `--postgresql-missing-dictionaries`: Set `postgresql.missing_dictionaries`
+  ([docs][doc-settings]) in the Zulip settings, which alters the initial database.
+  This should be used with cloud managed databases like RDS. This option conflicts
+  with `--no-overwrite-settings`.
+
+* `--no-init-db`: This option instructs ther installer to not do any database
+  initialization. This should be used when you already have a Zulip database.
+
+* `--no-overwrite-settings`: This option preserves existing `/etc/zulip`
+  configuration files.
+
+* `--no-dist-upgrade`: This option makes the installer skip the initial
+  `apt-get dist-upgrade`.
```

Comment:
Agree.

Also worth noting _why_ one might want to pass `--no-dist-upgrade`, in addition to what it does.  I believe we only use it when starting up the one-click app in DO.  I'm not entirely convinced by the logic of doing so, which seems a like it might cause the system to miss important security patches, in the name of getting the instance up more quickly.

Reply:
Oh, and this documentation (wherever it goes) should also include the list of values for `--postrgesql-version` that are supported.

Reply:
@alexmv can you provide that list as not sure where I'd find it or what we specifically support.

Reply:
https://zulip.readthedocs.io/en/stable/production/postgres.html#postgres-database-details

Reply:
We should also make sure this change doesn't race with #18271.